uniro_frontend/src/App.tsx

```
@@ -24,22 +26,25 @@ function App() {
+					<Routes>
+						<Route path="/demo" element={<Demo />} />
+						<Route path="/" element={<LandingPage />} />
+						<Route path="/university" element={<UniversitySearchPage />} />
+						<Route path="/building" element={<BuildingSearchPage />} />
+						<Route path="/map" element={<MapPage />} />
+						<Route path="/form" element={<ReportForm />} />
+						<Route path="/result" element={<NavigationResultPage />} />
+						<Route path="/report/route" element={<ReportRoutePage />} />
+						<Route path="/report/risk" element={<ReportRiskPage />} />
+						/** 에러 페이지 */
+						<Route path="/error" element={<ErrorPage />} />
```

Biome (1.9.4)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)